public/history.php

```diff
@@ -230,8 +241,13 @@ function resize() {
         $memberSince = !empty($userSignedUp) ? getNiceDate(strtotime($userSignedUp), true) : '';
         echo "Member since: " . $memberSince . "<br>";
         echo "<br>";
-        echo "<br>";
-        echo "<br>";
+        echo "<form name='gotodateform' action='/historyexamine.php' onsubmit='convertDate()'>";
+        echo "<label for='d'><b>Jump to Date: </b></label>";
```

Comment:
```suggestion
        echo "<label for='d'><span class='font-bold'>Jump to Date: </span></label>";
```

I believe there is a push to get away from using these `<b>` tags in favor of spans with Tailwind classes. (also a similar push elsewhere to mitigate `<br>` usage)

ref: https://tailwindcss.com/docs/font-weight